pkgs/misc/ite-backlight/default.nix

```diff
@@ -0,0 +1,46 @@
+{ lib
+, pkgs
+, stdenv
+, ninja
+, libusb1
+, meson
+, boost
+, fetchFromGitHub
+, pkg-config
+, cpp-gsl }:
+
+stdenv.mkDerivation rec {
+  pname = "ite-backlight";
+  version = "v1.1";
+
+  src = fetchFromGitHub {
+    owner = "hexagonal-sun";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256:1hany4bn93mac9qyz97r1l858d48zdvvmn3mabzr3441ivqr9j0a";
+  };
+
+  nativeBuildInputs = [
+    ninja
+    pkg-config
+    meson
+    cpp-gsl
+    boost
```

Comment:
Sure this belongs not into buildInputs?

Reply:
Yep; `boost` is only needed for it's `string` template library.  I believe all the code gets instantiated at compile time so there is no run-time dependency.

Reply:
buildInputs is not about runtime dependencies. It is used for cross compiling and I think boost and all other libraries need to go buildInputs.

Reply:
Agreed.  Moved to `buildInputs`.